src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-agent-gradle.mdx

```diff
@@ -50,131 +50,172 @@ To install the Android agent, you need to use our [guided install](https://onenr
+
+In this example, `AGENT_VERSION` represents your agent version number. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes), and use the latest version.
+
+Starting with version 7.6.0, the New relic Android Gradle plugin is available on the [Gradle Plugin Portal](https://plugins.gradle.org/). This greatly simplifies the resolution and loading of community plugins.
+
+Newer apps should follow the instructions to [Apply the plugin using the plugins DSL block](#plugin-dsl) to use agent plugin hosted on the Gradle Plugin Portal. 
+
+If adding the New Relic Android agent to an older app, follow the instructions to [apply the plugin using the buildscript](#buildscript). For guidance on different approaches to resolving plugins, see [Resolving plugins](https://docs.gradle.org/current/userguide/plugins.html#sec:binary_plugin_locations).
+
+
+1. Set app permissions. Update your `AndroidManifest.xml` file to allow your app to access the internet and check network state:
+
+   ```xml
+   <uses-permission android:name="android.permission.INTERNET" />
+   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+   ```
+
+2. Start the Android agent. In your main Activity class (`onCreate` method), import the `NewRelic` class:
+
+   ```java
+   import com.newrelic.agent.android.NewRelic;
+   ```
+
+   <Callout variant="caution">
+    Starting the agent in other classes is not supported.
+   </Callout>
+
+3. Replace `GENERATED_TOKEN` with your actual app token (obtained during guided install) and add this code:
+
+   ```java
+   NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplicationContext());
+   ```
+
+4. (Optional) Configure for minification:
+
+    1. If you use ProGuard or Dexguard for code shrinking, create a `newrelic.properties` file in your app-level directory (`_projectname/app_`). 
+    2. Add this line to the file, replacing `GENERATED_TOKEN` with your actual token:
+        ```ini
+        com.newrelic.application_token=GENERATED_TOKEN
+        ```
+    3. To finish set up for minification, follow the steps in [configure ProGuard or DexGuard for Android apps](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps).
+
+5. Clean your project, then run your app in an emulator or device to generate traffic. Wait a few minutes as your agent captures that data.
+
+6. View your app's data on New Relic by going to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app)**</DNT>.
+
+
+## Configure the Android agent for Gradle Plugin Portal [#configure-plugin-portal]
+
+Starting with agent version 7.6.0, the Android agent plugin is available on the [Gradle Plugin Portal](https://plugins.gradle.org/) as a community plugin. The Gradle plugin DSL simpifies adding plugin dependencies to apps. 
 
    <CollapserGroup>
+
```

Comment:
@ally-sassman This block is of snippets is a duplication of what is provided below. in #apply-plugin-dsl and #buildscript sections. I left it here so Docs could decide on the best format.